### PR TITLE
Phase2-hgx121 Add a second scenatio D30 with C5 version for calorimeters

### PR DESCRIPTION
**Pull request by bsunanda**
C4 (Yes) version differs from C5 (No) version in the use of virtual wafers

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by boudoul** [comment]
Dear @civanch , @kepdro88, @bsunanda this is going to conflict to the PR I opened beforehand #23720

**Comment by kpedro88** [comment]
yes, @bsunanda please increment this to D30

File: Configuration/Geometry/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Tracker:
 Calorimeters:
 * C3: HGCal (v8) + Phase2 HCAL and EB
 * C4  HGCal (v9) + Phase2 HCAL and EB
+* C5  HGCal (v9) + Phase2 HCAL and EB (without virtual wafers)
```
Comment (kpedro88) [inline Configuration/Geometry/README.md]: `C5: HGCal (v9, without virtual wafers) + Phase2 HCAL and EB`

Comment (bsunanda) [reply Configuration/Geometry/README.md]: Done

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
@bsunanda please also update the PR title

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
@bsunanda @kpedro88 now that scenario D29 has been merged, this PR need to be rebased before its merge

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by bsunanda** [comment]
Closing this and a new PR #23773 with similar content is opened

**Pull request closed without merging**